### DISCUSSION TITLE
Way to edit content of outgoing emails?

### DISCUSSION THREAD

**Discussion by MaryMuse** [mastodon/mastodon#27226] [Q&A]
Hello,
I couldn't find any information using a search engine, but is there a way to edit the content of the outgoing emails. I would love to add a line at the bottom for my outgoing server emails that says "For support please email (blah blah blah) and then with a plug to support us on Patreon/Liberapay"

Is there way to do this in the interface or within the program itself? Thank you!!!!

**Comment by mjankowski** [comment]
I dont think there's a way to do this (without modifying source code)

**Reply by johnniemarbles** [reply]
I am also interested in this
